src/Tools/dotnet-user-jwts/test/UserJwtsTests.cs

```diff
@@ -631,7 +638,19 @@ public void List_CanHandleNoProjectOptionProvided_WithNoProjects()
         var app = new Program(_console);
         app.Run(new[] { "list" });
 
-        Assert.Contains("No project found at `-p|--project` path or current directory.", _console.GetOutput());
+        Assert.Contains($"Could not find a MSBuild project file in '{Directory.GetCurrentDirectory()}'. Specify which project to use with the --project option.", _console.GetOutput());
+    }
+
+    [Fact]
+    public void List_CanHandleProjectOptionAsPath()
+    {
```

Comment:
I think this is also unrelated to the project change?

Reply:
It is. Previously, we did not support searching for projects given a directory, we expected a complete filepath to a csproj. This allows querying with a filepath as the project options. The test name could probably be a little clearer.